CODEOWNERS

```diff
@@ -4,7 +4,7 @@
 /charts/argo @stefansedich @paguos @vladlosev @yann-soubeyrand
 
 # Argo CD
-/charts/argo-cd @seanson @spencergilbert @davidkarlsen @mr-sour @yann-soubeyrand
+/charts/argo-cd @seanson @spencergilbert @davidkarlsen @mr-sour @yann-soubeyrand @abdennour
```

Comment:
`CODEOWNER` changes are done typically by one of the ArgoCD owners (such as https://github.com/argoproj/argo-helm/pull/551). if you'd like to volunteer to be a contributor please give a shout in the #argo-helm Slack channel.

Reply:
> Looks good but needs the CODEOWNERS change removed.
Done! I removed it.
@seanson @vladlosev  kindly proceed